program/tests/duplicate_block_proof.rs

```diff
@@ -124,6 +135,58 @@ async fn write_proof(
     }
 }
 
+async fn close_report(
+    context: &mut ProgramTestContext,
+    report_key: Pubkey,
+    destination: Pubkey,
+    initial_lamports: u64,
+    increment: Epoch,
+) -> Result<(), BanksClientError> {
+    increment_clock(context, increment).await;
+    assert!(context
+        .banks_client
+        .get_account(destination)
+        .await
+        .unwrap()
+        .is_none());
```

Comment:
It might be worth adding a test for when the destination is the report account, which means this check won't always be valid